README.md

```diff
@@ -1,4 +1 @@
-paper-radio-group
-=================
-
-See the [component page](http://www.polymer-project.org/docs/elements/paper-elements.html#paper-radio-group) for more information.
+# paper-radio-group
```

Comment:
I don't think we have really decided what to do about this for 0.8. I have to admit, I would prefer that the README contain documentation that we currently put in the leading comment of the element's import.

Do you have any thoughts about this, @morethanreal ?